tests/e2e/tests_runner.sh

```diff
@@ -107,6 +107,11 @@ main() {
 
 	cd "${tests_repo_dir}/integration/kubernetes/confidential"
 
+        # Update runtimeclass and kata default config for the scripts in tests repo
+        sed -i "s/runtimeClassName: kata/runtimeClassName: $runtimeclass/g" fixtures/*.yaml
```

Comment:
@arronwy sent a PR to address that on tests side: https://github.com/kata-containers/tests/pull/5131 . Notice that replacement with sed doesn't work for the tests in `agent_image_encrypted.bats` because the pod configuration is fetched from an external repository.